### ISSUE TITLE
Afk bug, Others unafk after someone else does even though they are still afk

### ISSUE THREAD

**Issue by AutumnUwU** [Zrips/CMI#6743]
**Description of issue:**
Whenever someone comes back from afk it randomly takes off afk from someone else as well constantly. It is a bug as it has no joke done this over 100 times, Whenever someone comes back from afk EVERY time it unafk's someone else too when they are still afk. 
![image](https://user-images.githubusercontent.com/84705226/158900349-7a383730-5dd3-4cfb-9579-ad0263aa8f78.png)

---

---

**Cmi Version (using`/cmi version`):** 
9.1.3.0
**Server Type (Spigot/Paperspigot/etc):** 
Paper
**Server Version (using `/ver`):** 
Paper 214 1.18.1 (latest release of 1.18.1 paper)

**Comment by Zrips** [comment]
Will be fixed with next upadte

**Comment by Zrips** [comment]
Afk toggle should not have any impact on another player afk status. Did you got any error in console when that happen?

**Comment by AutumnUwU** [comment]
> Afk toggle should not have any impact on another player afk status. Did you got any error in console when that happen?

No, I do not get any errors in console when it happens, I just know whenever me or another player comes back from being afk from talking in chat or moving etc. it most the time unafk's someone else who is still afk and hasn't moved or talked, on top of unafking the person that isn't afk anymore, Not really sure whats causing it.
